Makefile

```diff
@@ -615,7 +622,7 @@ realclean: clean java_clean
 	@$(RM) -f *.bin debian/changelog*
 
 ### additionally removes generated files not in the repository tree
-distclean: realclean clean_genRules
+distclean: clean_stack realclean clean_genRules
```

Comment:
Can't we auto-generate some debian package from the Haskell packages installed by Stack? And then use this debian package for the PPA build process? We would need a flag that indicates whether we are in the PPA build process or not.